### ISSUE TITLE
Access to ScreencastIface causes warning in logs

### ISSUE THREAD

**Issue by Ian2020** [EasyScreenCast/EasyScreenCast#277]
## Expected Behavior

Clean logs.

## Current Behavior

The following appears in the logs:

```
Some code accessed the property 'ScreencastIface' on the module 'screencast'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
```

## Possible Solution

@somethingGoneWrong suggested a possible solution in #272. Gnome-shell maintainers commented [also](https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3238#note_928180) that `loadInterfaceXML('org.gnome.Screencast')` should be used and added: "As a bonus, that change will also work on 3.38, while the current code breaks (because screencasting was split out into a separate process)."

## Steps to Reproduce (for bugs)

1. Enable the extension, start a GNOME session
2. Check your logs: `journalctl -b0 | grep ScreencastIface`

## Logs

Above.

## Your Environment

* Where did you download the extension?
 [ ] Gnome shell extension website
 [X] From Git (3252312c408e687d1f89342951761e1df8536828)
* Gnome shell version: 3.36.6

* Operating System and version: Fedora 32

* Display server: X11

**Comment by Ian2020** [comment]
Changing the code at the top of utilrecorder.js to this resolves the issue:

```
...
const { loadInterfaceXML, loadSubInterfaceXML } = imports.misc.fileUtils;
const ScreencastIface = loadInterfaceXML('org.gnome.Shell.Screencast');
...
const ScreenCastProxy = Gio.DBusProxy.makeProxyWrapper(
    ScreencastIface
);
...
```

...however @somethingGoneWrong said this could be hacky. I can submit PR though if this is suitable solution?

**Comment by Ian2020** [comment]
I think this has been resolved by merging #276
